Detectors/ITSMFT/ITS/tracking/GPU/cuda/CMakeLists.txt

```diff
@@ -50,10 +50,11 @@ o2_add_library(ITStrackingCUDA
                                      O2::SimulationDataFormat
                                      O2::ReconstructionDataFormats
                                      CUDA::nvToolsExt # TODO: change to CUDA::nvtx3 when CMake bump >= 3.25
+               PRIVATE_LINK_LIBRARIES O2::GPUTrackingCUDAExternalProvider
```

Comment:
This links the external CUDA object library, with TrackParCov and O2Propagator in device-linkable form.
So you have to compile your code with CUDA_SEPARABLE_COMPILATION, and then everything is automatically in by linking this library.

The library also instantiates a static object which registers at GPU reconstruction and passes the pointer for its constant buffer. So when GPU reconstruction update the constant memory, also this instance will be updated automatically.